pom.xml

```diff
@@ -1104,6 +1104,12 @@
                 <module>tests/ballerina-test-integration</module>
             </modules>
         </profile>
+        <profile>
+            <id>benchmark</id>
```

Comment:
Shall we use "benchmarks" as the profile name since we run multiple benchmarks?

Reply:
fixed